portfolio/src/main/webapp/index.html

```diff
@@ -7,68 +7,77 @@
     <script src="script.js"></script>
   </head>
   <body>
-	  <p>Click <a href="/data">here</a> to view content generated by a servlet.</p>
-    <div id="content">
+	<p>Click <a href="/data">here</a> to view content generated by a servlet.</p>
```

Comment:
nit, this is over-indented by 2 spaces

Reply:
This isn't indented on my device so maybe it just looks over indented on github